engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -1390,10 +1390,12 @@ else if (volumeInfo.getFormat() == ImageFormat.OVA) {
 
                 verifyCopyCmdAnswer(copyCmdAnswer, templateInfo);
 
+                // Seems like vmware 6.5 and above does not automatically remount the datastore after rescanning the storage.
+                // Not sure if this is needed.
                 // If using VMware, have the host rescan its software HBA if dynamic discovery is in use.
-                if (HypervisorType.VMware.equals(templateInfo.getHypervisorType())) {
```

Comment:
cc @harikrishna-patnala @nvazquez could this cause any regression?

Reply:
@skattoju4 There are two more methods in which disconnectHostFromVolume() method is being called. Any specific reason why you want to remove this method call only here. Can you check the other two methods as well handleCreateManagedVolumeFromManagedSnapshot() and handleCopyAsyncToSecondaryStorage(). These two are also related to managed storage.